### ISSUE TITLE
Bizarre "FATAL" On My End.

### ISSUE THREAD

**Issue by adamesq** [rrirower/Channel-Feed-Manager#225]
**Describe the bug**
As a Mac guy, I've been running CFM on a Windows server vis AWS. I've just switched over to UTM (so cool, BTW!). I re-ingested my current feed, but on launch I get a supposedly FATAL error claiming "Do Not Show json" is invalid. Unlike Mortal Kombat, this fatality doesn't appear to be fatal at all. The app goes on to function properly. I tried reinstalling, but I'm still getting the error. 

**To Reproduce**
1. Launch CFM

**Expected behavior**
No FATAL error.

**Screenshots**
<img width="1334" alt="Channel Feed Manager Issue" src="https://github.com/user-attachments/assets/92eb1e38-c9b4-4156-88e0-cdada667e879">

**Version number:**
 - Version 1.0.1.6

**Additional context**
Thinking there's an easy fix here, but 🤔

**Comment by rrirower** [comment]
Does the path "C:\Users\Adam\Documents\Channel Feed Manager\Temp" exist on your install?

You are the only one experiencing this error.  I think this is isolated to your configuration on UTM.

**Comment by adamesq** [comment]
Yes, the path is there. Very odd. I don't *think* it's causing an issue other than the pop-up. Thank you.

**Comment by adamesq** [comment]
AHHH...The Temp folder is EMPTY. There is no "Do Not Show.json" in there.

**Comment by rrirower** [comment]
Do you have "write" access to the folder?

**Comment by adamesq** [comment]
<img width="354" alt="Screenshot 2024-10-07 at 12 12 36 PM" src="https://github.com/user-attachments/assets/a094db18-6046-4f72-9cfe-99c165c4b410">

**Comment by rrirower** [comment]
It appears you have read-only access to the files in the temp folder.  The software tries to write a file into that folder. You must have write access for that to work.

**Comment by adamesq** [comment]
It's letting me write and read. I tried removing the CFM folder and reinstalled again. I launched with a completely clean CFM. However, the error persists. I'll try adding to the feed and see if anything else arises later. I just want to be free of needing AWS. 😂 I'll report back just in case someone else runs into this odd situation.

**Comment by rrirower** [comment]
Do you have any feedback on this issue?

**Comment by rrirower** [comment]
No feedback.  Closed.
